src/lib/components/ratio/ColorIssues.svelte

```diff
@@ -53,42 +41,25 @@
         When a color is out of gamut for the user's screen, the browser will
         adjust the color to be in gamut.
       </p>
-    </dd>
-  </dl>
-</details>
+    </li>
+  </ul>
+</div>
 
 <style lang="scss">
-  @use 'config';
-  .known-issues {
-    grid-area: knownissues;
-    padding: var(--gutter);
-
-    @include config.above('lg-page-break') {
-      margin-inline: calc(var(--gutter) * -1);
-    }
+  hr {
+    border: unset;
+    border-block-start: var(--border-width) solid var(--border-light);
+    margin-block-end: var(--spacer);
+    margin-inline: unset;
```

Comment:
Would it be easier to change this to match the padding-inline value of everything in main, then I don't think we need the grid with empty columns?

`margin-inline: var(--double-gutter);`

I'm guessing that is why we changed `[data-layout~=main]`?

Reply:
I think I might've changed this when I thought we were keeping the details/summary triangle, making space for that to line up correctly. 

I fixed this in https://github.com/oddbird/oddcontrast/pull/223/commits/7e27f5eca78df035e0d1145ae0b81dd0346b27c2